element.go

```diff
@@ -342,29 +342,47 @@ func readRawItem(d *dicomio.Decoder) ([]byte, bool) {
 	return d.ReadBytes(int(vl)), false
 }
 
+// NativeFrame represents a native image frame
+type NativeFrame struct {
+	// Data is a slice of pixels, where each pixel can have multiple values
+	Data          [][]int
+	Rows          int
+	Cols          int
+	BitsPerSample int
+}
+
+// EncapsulatedFrame represents an encapsulated image frame
+type EncapsulatedFrame struct {
+	// Data is a collection of bytes representing a JPEG encoded image frame
+	Data []byte
+}
+
+// Frame wraps a single encapsulated or native image frame
+type Frame struct {
+	IsEncapsulated   bool
+	EncapsulatedData EncapsulatedFrame
```

Comment:
Field called `EncapsulatedData` to better represent the hierarchy of `Frame`s containing `EncapsulatedData` instead of `Frame`s containing other `EncapsulatedFrame`s...ideally we can come up with a better name for the struct too. Perhaps `EncapsulatedFrameData`?